### ISSUE TITLE
Create PWM Passthrough

### ISSUE THREAD

**Issue by benjohnemmett** [benjohnemmett/Wingman#6]
Create platform specific logic for ATMega4809 that outputs 4 PWM signals that follow the captured input. Use the Timer/Counter Type A. 

Test with ATMega4809 test board input capture connected to RC receiver and output connected to aircraft servos or logic analyzer.

\* This may be replaced later with external PWM control.

**Comment by benjohnemmett** [comment]
This was tested with an RC plane and was able to control elevator, rudder, ailerons, & propeller ESC. All four channels could not be tested with actual motors simultaneously without issues. It appears likely to be a current draw issue because the RC transmitter alarm sounded for low RX power. Even after disconnecting all but one motor, the MCU would not operate until it was restarted (done by reflashing the same program).

**Comment by benjohnemmett** [comment]
Further power testing pinpointed the power issue that was mentioned in the last comment to be in the jumper wires used to power the servos from the ESC. By just extending the 3-pin connection coming from the ESC to the RC receiver with a pair of jumper wires, it was enough to cause low power issues where there were none before. 

I may return to this testing with better jumper wires later on. For now, I measured the current draw of each servo motor independently to estimate the total power requirements of the servo power rail. I found they typically run 2.5 mA idle and 50 to 200 mA active which means that the servo power rail will need to support at least 800 mA for power drawn from the ESC connector to power the other 4 servos (rudder, elevator, 2x ailerons). I feel like that is enough to confidently move forward with the hardware design.
